include/deal.II/base/mpi.h

```diff
@@ -1463,6 +1482,38 @@ namespace Utilities
 #  endif
     }
 
+    template <typename T>
+    T
+    bcast(const MPI_Comm &   comm,
+          const T &          object_to_send,
+          const unsigned int root_process)
+    {
+#  ifndef DEAL_II_WITH_MPI
+      (void)comm;
+      (void)root_process;
+      return object_to_send;
+#  else
+      const auto n_procs = dealii::Utilities::MPI::n_mpi_processes(comm);
+      const auto my_rank = dealii::Utilities::MPI::this_mpi_process(comm);
```

Comment:
I don't think you actually need these variables, do you? But maybe it would be worthwhile adding an assertion that `0<=root_process<n_proces`.